src/teleop/jetson/jetson_teleop.py

```diff
@@ -133,6 +143,39 @@ def ra_mode_callback(self, msg: String) -> None:
         with self._arm_mode_lock:
             self._arm_mode = msg.data
 
+    def brushless_encoder_callback(self, msg: MotorsStatus) -> None:
+        """
+        Callback for brushless encoder values
+        :param msg: MotorsStatus message representing the current status of the brushless motors
+        :return:
+        """
+        with self._joint_state_lock:
+            for i, name in enumerate(msg.name):
+                index_of_joint = self.current_ra_joint_states.name.index(name)
+                self.current_ra_joint_states.position[index_of_joint] = msg.joint_states.position[i]
+                self.current_ra_joint_states.velocity[index_of_joint] = msg.joint_states.velocity[i]
+                self.current_ra_joint_states.effort[index_of_joint] = msg.joint_states.effort[i]
```

Comment:
Just to make sure, the incoming message will have a fully populated list of effort values even if we're not using it, right?

Reply:
Yep, especially will send a joint state msg will all of the arrays being the same size